components/camel-salesforce/camel-salesforce-component/src/test/java/org/apache/camel/component/salesforce/CompositeApiBatchIntegrationTest.java

```diff
@@ -155,7 +155,9 @@ public void shouldSupportLimits() {
         final Map<String, String> apiRequests = (Map<String, String>) limits.get("DailyApiRequests");
 
         // for JSON value will be Integer, for XML (no type information) it will be String
-        assertEquals("15000", String.valueOf(apiRequests.get("Max")));
+        // This number can be different per org, and future releases,
+        // so let's just make sure it's greater than zero
+        assert(Integer.valueOf(String.valueOf(apiRequests.get("Max"))) > 0);
```

Comment:
Please do not use Java **assert** keyword. Java Assertions are disabled by default in maven-surefire. I am pretty sure that your test will pass even with `assert(false)`. Use JUnit method assertTrue instead.

Reply:
ok